sap/5.3/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -11,12 +11,10 @@ NOTE: Starting with version 5.0, SAP Connector also supports connecting to an on
 == Limitations
 
 * SNC is not supported for use in Mule apps that deploy SAP Connector in CloudHub and Anypoint Runtime Fabric.
-* Multiple applications with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST deployed on the same Mule runtime engine does not work because the JCoLibrary does not allow the creation of two servers connecting to instances that have the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST. If you attempt to create two servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST, only one server is created. +
-Workarounds:
-. Use two different Mule runtime engines.
-. Use the same PROGRAM_ID but with two different GATEWAY_HOSTs.
-
-* SAP Connector does not support Bitronix Transaction Manager. 
+* For versions prior to SAP Connector 5.3.0, multiple applications deployed on the same Mule runtime engine with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST cannot be created by the JCo library. If you attempt to create two servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST, only one server is created. Some workarounds are to use two different Mule runtime engines or use the same PROGRAM_ID with two different GATEWAY_HOSTs.
```

Comment:
```suggestion
* For SAP Connector versions earlier than 5.3.0, multiple applications deployed on the same Mule runtime engine with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST cannot be created by the JCo library. If you create two servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST, only one server is created. To work around this, you can use two different Mule runtime engines or use the same PROGRAM_ID with two different GATEWAY_HOSTs.
```

Reply:
So, there is a mix of application and server.  Is this really:
For versions prior to SAP Connector 5.3.0, multiple servers with the same PROGRAM_ID, GATEWAY_SERVICE, and GATEWAY_HOST cannot be created by the JCo library. 
?